lighthouse-core/lib/i18n/i18n.js

```diff
@@ -174,7 +174,23 @@ const _icuMessageInstanceMap = new Map();
  */
 function _formatIcuMessage(locale, icuMessageId, icuMessage, values) {
   const localeMessages = LOCALES[locale];
-  const localeMessage = localeMessages[icuMessageId] && localeMessages[icuMessageId].message;
+  let localeMessage = localeMessages[icuMessageId] && localeMessages[icuMessageId].message;
+  // lets try to replace some things :o
+  const placeholders = localeMessages[icuMessageId] && localeMessages[icuMessageId].placeholders;
+  if (placeholders) {
+    console.log('placeholders!');
+    // do some regex
+    Object.entries(placeholders).forEach(entry => {
+      const key = entry[0];
+      const value = entry[1];
+      //use key and value here
+      let regexStr = `\$${key}\$`;
+      console.log(key, value, regexStr);
+      localeMessage = localeMessage.replace(regexStr, value.content);
```

Comment:
maybe to avoid the issue with positional replacements (`$1` etc), we could use use the named replacements in `values` if `value.content` is `\$\d+` or whatever? We'll just have to figure out how to do the mapping (just use `key`?)